packages/core/src/routes-me/social.ts

```diff
@@ -131,7 +123,7 @@ export default function socialRoutes<T extends AuthedMeRouter>(
       });
 
       assertThat(
-        !(await hasUserWithIdentity(target, socialUserInfo.id)),
+        !(await hasUserWithIdentity(target, socialUserInfo.id, userId)),
```

Comment:
should we handle this in frontend instead of backend?

Reply:
This is to detect if this current social account has been linked to another Logto account. Therefore we cannot handle it in frontend